wazo_call_logd/plugins/cdr/plugin.py

```diff
@@ -13,13 +13,13 @@
 from .service import CDRService
 
 
-class Plugin(object):
+class Plugin:
     def load(self, dependencies):
         api = dependencies['api']
         config = dependencies['config']
 
         auth_client = AuthClient(**config['auth'])
-        dao = CallLogDAO(new_db_session(config['db_uri']))
+        dao = DAO(new_db_session(config['db_uri'])).call_log
```

Comment:
An instance of `DAO`, which contains an instance of all DAOs is created here and in the support-center plugin.py
dao should be a dependency that is injected in the dependencies

Reply:
:+1: I created a new issue to refactor this part https://wazo-dev.atlassian.net/browse/WAZO-2021